docs_guide/sphinx_guide/how_to_build_docs.rst

```diff
@@ -0,0 +1,103 @@
+##################################
+How to build documentation locally
+##################################
+
+.. include:: ../others/how-to_type.rst
+
+This guide shows how to build the documentation of a Qiskit project repository in local.
+
+Pre-requisites
+==============
+
+This guide assumes your Qiskit project already has a working Sphinx documentation project in a folder called ``docs``. If you don't have it, you can set it up with
+:doc:`sphinx-quickstart <sphinx:man/sphinx-quickstart>`.
+
+Install package and dependencies
```

Comment:
Should we change the narrative of this guide from Qiskit package to Ecosystem like we did with the new doc project tutorial?

Reply:
I was thinking "Qiskit project" refers to projects in the ecosystem but I agree that is not explicit. Let me think about how to rephrase that.

Reply:
Fixed in https://github.com/Qiskit/qiskit_sphinx_theme/pull/173/commits/f714271240c3ebd1fa2215862081f3522e49380a